### ISSUE TITLE
Incompatibility with DHAVE_OPENMP=OFF

### ISSUE THREAD

**Issue by sylm21** [neperfepx/neper#685]
**Describe the bug**

Error during linking on ubuntu with wsl 

The file neut/neut_mesh/neut_mesh_op/neut_mesh_op.c

Doesn't have flag compilation in line 2048. I have add this and it's ok

```cpp
#ifdef HAVE_OPENMP
#pragma omp parallel
  if (omp_get_thread_num() != 0)
  {
    printf ("\nneut_mesh_init_simeltori is not thread-safe!\n");
    ut_print_neperbug ();
  }
#endif
```

Thanks

**Comment by rquey** [comment]
Fixed in version 4.6.1-1. Thanks @sylm21.

**Comment by sylm21** [comment]
Thanks
